splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -525,6 +525,8 @@ def generate(
         global_config = global_config_lib.GlobalConfig(
             config_path, is_global_config_yaml
         )
+        # handle the update of globalConfig before validating
+        global_config_update.handle_global_config_update(global_config)
```

Comment:
This is a potential breaking change, right?

Reply:
The change is around the usage of `activeResponse` (wrong usage) vs `adaptiveResponse` (right usage) in globalConfig. In order to correctly validate the globalConfig, I have removed the `activeResponse` attribute from schema.json. If we don't remove it from schema.json and keep it for backward compatibility, it would imply that both activeResponse and adaptiveResponse are valid, yet the UCC framework changes it to adaptiveResponse runtime.

Hence, to handle the change of usage (and similar any other breaking changes that we may have to do in future), we update the globalConfig first (handle all the changes), then validate it and then parse it to build the code.

Reply:
Sounds good.